utils/build-script-impl

```diff
@@ -740,20 +743,26 @@ CROSS_TOOLS_DEPLOYMENT_TARGETS=()
 
 # Determine the native deployment target for the build machine, that will be
 # used to jumpstart the standard library build when cross-compiling.
-case "$(uname -s)" in
-    Linux)
+# FIXME: This will not work while trying to cross-compile
+# It gets the architecture from the system processor arch.
```

Comment:
Excellent point, you're right.  I've removed the comment.  It's not collapsing this change in the pull request view, because I moved this development into the armv7 branch, and it's unclear to me how PRs work with branches that only exist on one fork.